trillian/ctfe/ct_server/main.go

```diff
@@ -140,10 +141,19 @@ func main() {
 			etcdRes.Update(ctx, *etcdMetricsService, byeMetrics)
 		}()
 	} else if strings.Contains(*rpcBackend, ",") {
-		glog.Infof("Using FixedBackendResolver")
-		// Use a fixed endpoint resolution that just returns the addresses configured on the command line.
-		res := util.FixedBackendResolver{}
-		dialOpts = append(dialOpts, grpc.WithBalancer(grpc.RoundRobin(res)))
+		// This should probably not be used in production. Either use etcd or a gRPC
+		// load balancer. It's used by the integration tests.
+		glog.Warningf("Multiple RPC backends from flags not recommended for production.")
```

Comment:
Seems that you don't need the `f` variant since there are no params?

Reply:
Might also be worth adding the recommendation text from the comment to the logged warning too - save folks from having to dig in the code to know what to do instead?

Reply:
Removed the 'f' and expanded the comment.